prisma/schema.prisma

```diff
@@ -67,3 +67,18 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+// Profile schema
+enum Role {
+  ADMIN
+  MENTOR
+  MENTEE
+}
+
+model Profile {
+    user            User    @id @unique
```

Comment:
Can you take a look at this https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-one-relations and do the one-to-one relation like this? I don't think this will work.

Also, make sure you are running `npm run db:push` to ensure that the schema is valid. There is a VSCode extension for Prisma schema and it gives errors about invalid schemas.